src/pybind/mgr/mgr_util.py

```diff
@@ -1,6 +1,15 @@
 import contextlib
 import socket
 
+try:
+    from more_itertools import pairwise
```

Comment:
Does this lead to a new Python package dependency? Or are these modules part of Python core?

Reply:
I'd prefer to have this as a real dependency in octopus, indeed.

Reply:
We used them in the dashboard so I assumed we could use them here too, without further declaration, am I wrong with this assumption?

Reply:
right now, it doesn't add another hard dependency, no.